src/app/mandate/mandate.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component } from '@angular/core';
+import {ComponentsModule} from "@biggive/components-angular";
+import {DatePipe} from "@angular/common";
+import {ExactCurrencyPipe} from "../exact-currency.pipe";
+import {FaIconComponent} from "@fortawesome/angular-fontawesome";
+import {MatProgressSpinner} from "@angular/material/progress-spinner";
+import {Mandate} from "../mandate.model";
+
+@Component({
+  selector: 'app-mandate',
+  standalone: true,
+  imports: [
+    ComponentsModule,
+    DatePipe,
+    ExactCurrencyPipe,
+    FaIconComponent,
+    MatProgressSpinner
+  ],
+  templateUrl: './mandate.component.html',
+  styleUrl: './mandate.component.scss'
+})
+export class MandateComponent {
+  personId = 'donor-id';
+  complete = true;
+  encodedShareUrl = '';
+  encodedPrefilledText: string = '';
+  donation: any = {};
```

Comment:
we filled this in quickly to get the template working as it started, as a copy of the donate thanks template, but I think it needs more work to get it ready to merge.

E.g. this line is donation related not mandate, and we should avoid the `any` type.

I think quite a few other properties here may be not relevant to mandates.

Reply:
updated!